snmp-config.yml.sample

```diff
@@ -12,8 +12,8 @@ instances:
   - name: <INSTANCE IDENTIFIER>
     command: metrics
     arguments:
-      hostname: localhost
-      port: 161
+      snmp_host: localhost
```

Comment:
why are you changing this? I understand all integrations will have to change this config values as well?